src/Catalyst.Core.Modules.Consensus.Tests/UnitTests/Deltas/DeltaBuilderTests.cs

```diff
@@ -113,220 +117,227 @@ public void BuildDeltaEmptyPoolContent()
-            transactionRetriever.GetMempoolTransactionsByPriority().Returns(invalidTransactionList);
-
-            var deltaBuilder = new DeltaBuilder(transactionRetriever, _randomFactory, _hashProvider, _peerSettings,
-                _cache, _dateTimeProvider, _logger);
-            var candidate = deltaBuilder.BuildCandidateDelta(_previousDeltaHash);
-
-            ValidateDeltaCandidate(candidate, _zeroCoinbaseEntry.ToByteArray());
-
-            _cache.Received(1).AddLocalDelta(Arg.Is(candidate), Arg.Any<Delta>());
-        }
-
-        [Fact]
-        public void BuildDeltaCheckForAccuracy()
-        {
-            var transactions = Enumerable.Range(0, 20).Select(i =>
-            {
-                var transaction = TransactionHelper.GetPublicTransaction(
-                    (uint)i,
-                    receiverPublicKey: i.ToString(),
-                    transactionFees: (ulong)_random.Next(),
-                    timestamp: _random.Next(),
-                    signature: i.ToString());
-                return transaction;
-            }).ToList();
-
-            var transactionRetriever = BuildRetriever(transactions);
-            var selectedTransactions = BuildSelectedTransactions(transactions);
-
-            var salt = BitConverter.GetBytes(_randomFactory.GetDeterministicRandomFromSeed(_previousDeltaHash.ToArray()).NextInt());
-
-            var rawAndSaltedEntriesBySignature = selectedTransactions.SelectMany(
-                t => t.PublicEntries.Select(e =>
-                {
-                    var publicEntriesProtoBuff = e;
-                    return new
-                    {
-                        RawEntry = publicEntriesProtoBuff,
-                        SaltedAndHashedEntry =
-                            _hashProvider.ComputeMultiHash(publicEntriesProtoBuff.ToByteArray().Concat(salt))
-                    };
-                }));
-
-            var shuffledEntriesBytes = rawAndSaltedEntriesBySignature
-               .OrderBy(v => v.SaltedAndHashedEntry.ToArray(), ByteUtil.ByteListComparer.Default)
-               .SelectMany(v => v.RawEntry.ToByteArray())
-               .ToArray();
-
-            var expectedBytesToHash = BuildExpectedBytesToHash(selectedTransactions, shuffledEntriesBytes);
-
-            var deltaBuilder = new DeltaBuilder(transactionRetriever, _randomFactory, _hashProvider, _peerSettings,
-                _cache, _dateTimeProvider, _logger);
-            var candidate = deltaBuilder.BuildCandidateDelta(_previousDeltaHash);
-
-            ValidateDeltaCandidate(candidate, expectedBytesToHash);
-
-            _cache.Received(1).AddLocalDelta(Arg.Is(candidate), Arg.Any<Delta>());
-        }
-
-        private static IDeltaTransactionRetriever BuildRetriever(List<TransactionBroadcast> transactions)
-        {
-            transactions.ForEach(t => t.AfterConstruction());
+            //transactions.ForEach(t => t.AfterConstruction());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4562326)